specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-10-03-preview/HybridCompute.json

```diff
@@ -3010,6 +3473,124 @@
       ],
       "description": "Properties for the Machine ESU profile."
     },
+    "LicenseStatus": {
+      "readOnly": true,
+      "type": "string",
+      "enum": [
+        "Unlicensed",
+        "Licensed",
+        "OOBGrace",
```

Comment:
I generally caution against using acronyms, unless they are definitely well known. these might not quite fit that criteria.

Reply:
These're well-defined values from SoftwareLicensingProduct class. 
https://learn.microsoft.com/en-us/previous-versions/windows/desktop/sppwmi/softwarelicensingproduct